### PR TITLE
Update elu_backward signature

### PR DESCRIPTION
**Pull request by JackCaoG**
This is to fix https://github.com/pytorch/xla/issues/2693. We need to match the node signature change in https://github.com/pytorch/pytorch/pull/49272

### CODE REVIEW

**Review by jysohn23 — APPROVED** [review]

File: torch_xla/csrc/aten_xla_type.cpp [hunk 1]
```diff
@@ -1204,9 +1204,12 @@ at::Tensor& AtenXlaType::elu_(at::Tensor& self, at::Scalar alpha,
 
 at::Tensor AtenXlaType::elu_backward(const at::Tensor& grad_output,
                                      at::Scalar alpha, at::Scalar scale,
-                                     at::Scalar input_scale,
+                                     at::Scalar input_scale, bool is_result,
                                      const at::Tensor& output) {
```
Comment (H-Huang) [inline torch_xla/csrc/aten_xla_type.cpp:1208]: Ended up renaming this param to `self_or_result` as well

File: torch_xla/csrc/aten_xla_type.h [hunk 2]
```diff
@@ -322,7 +322,7 @@ class AtenXlaType {
 
   static at::Tensor elu_backward(const at::Tensor& grad_output,
                                  at::Scalar alpha, at::Scalar scale,
-                                 at::Scalar input_scale,
+                                 at::Scalar input_scale, bool is_result,
                                  const at::Tensor& output);
```
Comment (H-Huang) [inline torch_xla/csrc/aten_xla_type.h:326]: `self_or_result`

**Review by H-Huang — APPROVED** [review]
Sorry for the delay, [pytorch/pytorch#49272](https://github.com/pytorch/pytorch/pull/49272) has been merged now, so this is ready to go after the changes.

**Comment by JackCaoG** [comment]
@H-Huang Thanks for the update, I will wait for the test to finish and merge this change

File: torch_xla/csrc/aten_xla_type.cpp [hunk 3]
```diff
@@ -1204,9 +1204,12 @@ at::Tensor& AtenXlaType::elu_(at::Tensor& self, at::Scalar alpha,
 
 at::Tensor AtenXlaType::elu_backward(const at::Tensor& grad_output,
                                      at::Scalar alpha, at::Scalar scale,
-                                     at::Scalar input_scale,
+                                     at::Scalar input_scale, bool self_or_result,
```
Comment (H-Huang) [inline torch_xla/csrc/aten_xla_type.cpp:1207]: I had meant `output` should be named to `self_or_result` and the bool param should remain `self`

**Pull request merged**